crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -14,18 +14,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+use arrow::array::RecordBatch;
 use arrow::compute::{filter_record_batch, SortOptions};
 use arrow::{
     array::{make_comparator, StructArray, TimestampNanosecondArray},
     datatypes::DataType,
 };
 use async_stream::stream;
 use cache::QueryResultsCacheProvider;
+use datafusion::physical_plan::stream::RecordBatchStreamAdapter;
```

Comment:
This can be merged with the import below

Reply:
Updated 👍